### ISSUE TITLE
Text overlap with the image and cannot be read

### ISSUE THREAD

**Issue by kramamur** [roseacademies/hedhihelp#15]
In both Lessons One and Two, there are images where the text overlaps with and cannot be read. Some of them are because the image color and text color are the same. In other cases, the "Quit Module" button sits over the text. We need to figure out a way to make the text appear in the empty space so that the context of the image is not lost, and the text can be read.

**Comment by Honora-W** [comment]
Yeah I noticed that when I made the text bigger last time, what's your suggestion?
